operator/src/test/java/oracle/kubernetes/operator/steps/ManagedServersUpStepTest.java

```diff
@@ -477,6 +489,46 @@ public void whenShuttingDown_allowAdminServerNameInListOfServers() {
     assertStoppingServers(createNextStepWithout("server2"), "server1", "server3", ADMIN);
   }
 
+  @Test
+  public void whenClusterStartupDefinedWithPreCreateServerService_addAllToServers() {
+    configureCluster("cluster1").withPrecreateServerService(true);
+    addWlsCluster("cluster1", "ms1", "ms2");
+
+    invokeStep();
+
+    assertThat(getServers(),
```

Comment:
Not sure why you have this on multiple lines. 

allOf( hasItem("ms1"), hasItem("ms2") )

doesn't take up that much horizontal space, and seems a bit easier to read, I think. In fact, I suspect that the entire assert would fit comfortably on a single line.

Reply:
I don't see a change or response to this comment.

Reply:
Oops. I missed some occurrences. Should be fixed now. Thanks!